dima/react-movies/src/components/currentMovieGenreFilter/currentMovieGenreFilter.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+
+interface CurrentMovieGenreFilterProps {
+  genres: string[];
+}
+const CurrentMovieGenreFilter = ({ genres }: CurrentMovieGenreFilterProps) => {
+  return (
+    <span className="filter-genre">
+      {" "}
+      Films by{" "}
+      {genres.map((genre) =>
+        genres.indexOf(genre) !== genres.length - 1 ? genre + "&" : genre
```

Comment:
**genres.indexOf(genre) !== genres.length - 1** - You can make it easier, just use second map parameter. `genres.map((genre, index) => index !== genre.length - 1`

Reply:
fixed